beekeeper-metadata-cleanup/src/main/java/com/expediagroup/beekeeper/metadata/cleanup/handler/ExpiredMetadataHandler.java

```diff
@@ -77,11 +78,18 @@ public void cleanupMetadata(HousekeepingMetadata housekeepingMetadata, LocalDate
       if (deleted && !dryRunEnabled) {
         updateAttemptsAndStatus(housekeepingMetadata, DELETED);
       }
+    } catch (BeekeeperIcebergException e) {
+      updateAttemptsAndStatus(housekeepingMetadata, SKIPPED);
+      log.warn("Table \"{}.{}\" is skipped because it is iceberg or could not be identified.",
```

Comment:
Why are you logging it twice?
If you decide to log it should not be a warning IMO. The Iceberg table is properly handled so there's no cause for alarm.

Reply:
I suggested because I didn't like the exception in the warn. So I suggested to put the exception in a debug just in case. We can make the warn an info if that's what you mean.

Reply:
Yes, `info` is what I'd go for. A `warn` makes me worry.